docs/triggers/trigger-basics.md

```diff
@@ -35,6 +40,27 @@ slack trigger create --trigger-def "path/to/trigger.ts"
 
 Example trigger objects in valid typescript and JSON formats can be viewed below.
```

Comment:
May want to re-iterate, as we do above, that once can use the generic to get input typeahead based on workflow definition inputs.